build/test-e2e-go/e2e.sh

```diff
@@ -30,7 +30,7 @@ echo "Tests took $(( end_time - start_time )) seconds"
 
 if [ -d "/logs/artifacts" ]; then
   echo "Creating junit xml report"
-  cat test_results.txt | go-junit-report > /logs/artifacts/junit_report.xml
+  cat test_results.txt | go-junit-report --subtest-mode=exclude-parents > /logs/artifacts/junit_report.xml
```

Comment:
This seems to make sense to me. I expect this means the parent results will disappear from the testgrid and we will ignore past failures, correct? Whereas if we used the `ignore-parent-results` option the parent result would show up as green?